shortcircuit-api/src/main/java/com/powsybl/shortcircuit/FaultResult.java

```diff
@@ -127,4 +158,13 @@ public Duration getTimeConstant() {
     public List<ShortCircuitBusResults> getShortCircuitBusResults() {
         return shortCircuitBusResults;
     }
+
+    /**
+     * The computation status. SUCCESS if computation went ok, NO_SHORTCIRCUIT if the transient reactance of every generators is missing,
```

Comment:
I would propose to move the doc for the different values to the enum class itself (each description goes to the javadoc of the corresponding enum value).

This way the user can refer to the definition of the values from anywhere they are used, not only from this getter.